Tests/FluentAssertions.Json.Shared.Specs/JTokenAssertionsSpecs.cs

```diff
@@ -432,7 +432,23 @@ public void When_checking_whether_a_JToken_is_not_equivalent_to_the_string_repre
             action.Should().Throw<XunitException>()
                 .WithMessage("Expected JSON document not to be equivalent*");
         }
-        
+
+        [Fact]
+        public void When_properties_contains_curly_braces_BeEquivalentTo_should_not_fail_with_FormatException()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            var actual = JToken.Parse(@"{ ""{a1}"": {b: 1 }}");
+            var expected = JToken.Parse(@"{ ""{a1}"": {b: 2 }}");
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act & Assert
+            //-----------------------------------------------------------------------------------------------------------
+            actual.Should().Invoking(x => x.BeEquivalentTo(expected))
```

Comment:
I would prefer to see the expected output instead of just the exception. For instance, would your fix change the output in any way? I'm not sure if there are existing tests that do that.

Reply:
added expected message